style.css

```diff
@@ -0,0 +1,762 @@
+  flex-flow: row wrap;
+  gap: 8px;
+  border-radius: 8px;
+  padding: 10px 0;
+  background-color: #f7f7f9;
+  margin-top: 10px;
+}
+
+.myself .lang .know .csss img {
+  margin-left: 5px;
+}
+
+.myself .lang .know .csss p {
+  color: #253858;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  margin-top: 18px;
+}
+
+.myself .lang .frame li {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 25px 0;
+}
+
+.myself .lang .skills li {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 25px 0;
+}
+
+.myself .lang .line {
+  width: 100%;
+  height: 1px;
+  background-color: #dfe1e6;
+}
+
+/* Contact Form */
+.co-form {
+  width: 100%;
+  height: auto;
+  background-image: url("image/contact-form-bg.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px 0 45px 0;
+  border-radius: 60px 0 0 0;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  margin-top: 114px;
+}
+
+.form-set {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 20px;
+  width: 327px;
+}
+
+.about-form {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.header-form {
+  text-align: center;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 53px;
+}
+
+.para-form {
+  text-align: center;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  color: #ebebff;
+  margin-top: 20px;
+}
+
+.text-input {
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  width: 327px;
+  padding: 10px 0 10px 15px;
+  margin-top: 20px;
+}
+
+.text-input::placeholder {
+  color: var(--neutral-n-800, #172b4d);
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  margin-right: 30px;
+  margin-left: 20px;
+}
+
+.contact-message {
+  width: 327px;
+  margin-top: 18px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  padding: 10px 0 100px 15px;
+}
+
+.contact-message::placeholder {
+  color: var(--neutrals-n-60, #b3bac5);
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.contact-btn {
+  width: 145px;
+  margin-top: 22px;
+  display: inline-flex;
+  padding: 12px 16px;
+  align-items: center;
+  border-radius: 8px;
+  border: none;
+  background: var(--neutrals-n-0, #fff);
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  color: var(--primary-b-400, #6070ff);
+}
+
+.about-all {
+  background: white;
+  margin-top: 114px;
+}
+
+/* Desktop Version */
+@media screen and (min-width: 768px) {
+  .logo {
+    margin-left: 71px;
+  }
+
+  .btn {
+    display: none;
+  }
+
+  .menu-sec {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    margin-right: 71px;
+    line-height: 50px;
+  }
```

Comment:
- Hey, let's kindly increase the spaces between each items and increase the space towards the end as seen in the images below to improve our design:




*Figma design*
![image](https://github.com/mar4444/re-portfolio-de/assets/59397159/8e6549d0-26b1-4987-8d47-6a40290024fb)


*your design*
![image](https://github.com/mar4444/re-portfolio-de/assets/59397159/3b215419-5742-4b3f-8efe-000582454fb6)